### ISSUE TITLE
amazing race generator

### ISSUE THREAD

**Issue by silky** [hahayes/hahayes-projects#15]
take a list of interesting things in a place; find related places maybe; then generate a tour of all the places, perhaps with puzzles to solve in order to find out how to get to the next place.

can be used in many ways - one where it is playing against you - i.e. it doesn't tell you where you are going; you need to solve the puzzles; or one where it tells you the answers and you can run the race for a bunch of people.

**Comment by mobeets** [comment]
haha this sounds fun! could also be seen as a scavenger hunt generator, where it uses local landmarks and gives you weird things to do at/with them, as a group

**Comment by silky** [comment]
yeah, good point! i wonder if people already do this with the geocaching thing?

it seems like a nice way to kind of do a random exploration of a bunch of 'interesting places/things' that we might have.

i quite like the idea of generating a random day tour of a place. especially in places that i don't know, where i get to configure things maybe.
